src/api/users.py

```
@@ -0,0 +1,99 @@
+    provider: str
+    roles: List[str]
+    permissions: List[str]
+    # OPENRAG_RBAC_ENFORCE — surfaced so the UI can hide RBAC-only
+    # sections (Users & Roles, Audit log, role pills) when the
+    # operator has the kill switch off.
+    rbac_enforced: bool
+
+
+@router.get("/me", response_model=MeResponse)
+async def get_me(
+    user: User = Depends(get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)